pkg/sentry/sentry.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 
 	"github.com/grafana/grafana-plugin-sdk-go/build"
+	"github.com/peterhellberg/link"	
```

Comment:
This repo only has 40 starts, and it doesn't seem well-maintained! There was only one commit in the last year.
I suggest not using it.

Reply:
This is the most popular repo I found. I think I can implement the method I need on my own. Do you think it worth it?

Reply:
Yes, I think it is a good idea to implement your own method. Thanks Oleg.

Reply:
I've checked the repo once again, and it seems that the entire code consists of just a 'Parse' method. So, if I decide to proceed with my implementation, I'll likely end up with a similar function. If you don't mind, I might leave this for further consideration later.

Reply:
Yes, that is fine. We can do it later.